hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/LazyDefiningSpecificMemberPms.java

```diff
@@ -113,11 +97,26 @@ public void init(@Nonnull Context context) throws Exception {
             }
         }
         if (address == null && !addresses.contains(address)) {
-            throw new JetException("Cluster does not contain the required member: " + memberAddress);
+            throw new JetException("Cluster does not contain the required member");
         }
         return addr -> supplier;
     }
 
+    @Override
+    public int preferredLocalParallelism() {
+        return 1;
+    }
+
+    @Override
+    public boolean isReusable() {
+        return true;
+    }
+
+    @Override
+    public boolean initIsCooperative() {
+        return true;
```

Comment:
cooperativeness depends on `partitionKeyExpr` cooperativeness

Reply:
javadoc does not say so, but `initIsCooperative` depends also on `PMS.get` being cooperative or not. In current implementation `init`+`get` are offloaded or not depending on `initIsCooperative`. Maybe worth fixing the javadoc, especially if we move expression evaluation to `get`

Reply:
javadoc: #24978